### DISCUSSION TITLE
The CW PROVIDENCE (WPRI)

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#766] [Q&A]
### Channel Name

THE CW PROVIDENCE

### Channel ID

WPRIDT1.us

### Website

https://www.wpri.com/the-cw-providence/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19761
